pinot-core/src/main/java/org/apache/pinot/core/data/recordtransformer/NullValueTransformer.java

```diff
@@ -22,28 +22,45 @@
 import org.apache.pinot.common.data.FieldSpec;
 import org.apache.pinot.common.data.Schema;
 import org.apache.pinot.core.data.GenericRow;
+import org.roaringbitmap.RoaringBitmap;
+
+import static org.apache.pinot.common.utils.CommonConstants.Segment.NULL_FIELDS;
```

Comment:
(minor) please don't use static import for non-testing class. Same for other classes